.snippets/code/mrl/polkadotxcm-send.js

```diff
@@ -0,0 +1,55 @@
+const moonbeamProvider = new WsProvider(MOONBEAM_CHAIN_WS_ENDPOINT);
+const moonbeamAPI = await ApiPromise.create({ provider: moonbeamProvider });
+
+const ethereumTx = await batchApproveTransferTx(moonbeamAPI);
```

Comment:
this is calling the function created in the evm-tx.js file, right?

Reply:
Yes. They're meant to be specific